### DISCUSSION TITLE
How can I make a newly created controller available for public access?

### DISCUSSION THREAD

**Discussion by Muratcol** [orchidsoftware/platform#1755] [Q&A] [answered]
I have a very simple problem. I created a new controller under `/app/Http/Controllers`.

And I defined route as `/fetch`

The controller works when I log in to the app however I want public access to this page. How can I do that?

**Comment by Muratcol** [comment, marked as answer]
@tabuna I'm searching for a solution for days but couldn't find one. What can I do to exclude an endpoint by a middleware? Can you help me, please?

**Reply by tabuna** [reply]
Hi @Muratcol. Find a route at a more suitable location. For example, `routes/web.php` or in `routes/api.php`.
Then the restrictions specified in the package configuration will not apply to it.

**Reply by Muratcol** [reply]
Wow, I can't believe I've been trying to fix this for days. I moved it to `routes/web.php` and the problem is gone. Thank you so much!
